go.work

```diff
@@ -70,6 +70,9 @@ use (
 	./internal/setup-smoke-test
 	./internal/traceprof/traceproftest
 	./orchestrion/all
+	./scripts/apiextractor
+	./scripts/autoreleasetagger
```

Comment:
@rarguelloF Why is this one required in the context of CI?

Reply:
Yeah, we shouldn't do `go work use -r .` anymore. We don't want to add tool dependencies in `go.work` anymore.